src/tools/populateLegacyOtherEntities.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-loop-func */
+import { Op } from 'sequelize';
+import { auditLogger } from '../logger';
+import { ActivityReport, ActivityRecipient, OtherEntity } from '../models';
+
+const hubOtherEntities = {
+  HSCO: 'Head Start Collaboration Office',
+  'Local/State Education System': 'State Education System',
+  'State Early Learning System / Guidelines': 'State Early Learning Standards',
+  'State Advisory Council': 'QRIS System',
+  'Regional TTA Team / Specialists': 'Regional TTA/Other Specialists',
+  'State Early Learning Standards / Guidelines':
+    'State Early Learning Standards',
```

Comment:
Should these some from a database? it just seems odd to have these values hardcoded.

Reply:
@kryswisnaskas correct me if I'm wrong, I was out when this script was written, but this script sets the `other-entity` value for legacy reports, and the label changed from the legacy reports to the labels we are using now. So legacy reports with an other-entity of 'State Advisory Council' were updated to have an other-entity of 'QRIS System'

Reply:
That's correct.